lib/setup

```diff
@@ -56,11 +56,13 @@ for i in "$@"
             --cron) CRON=1; shift;;
             --composer) COMPOSER=1; shift;;
 
-            *) ;;
+            *)
+                >&2 printf "\e[01;31mError: Unrecognized argument: %s\n" "${i}"
+                ;;
         esac
     done
 
-if [[ -z "$@" ]] || [[ "$HELP" == "1" ]]; then
+if [[ ! -z "$@" ]] || [[ "$HELP" == "1" ]]; then
```

Comment:
Hi David,

Thanks for your contribution!

It wouldn't break the execution of script with any parameters, even correct ones? It looks like it will always output usage text. 

Idea was to show usage on calling the script without parameters or with -h or --help flags.

Reply:
@shkoliar The loop immediately prior to this condition uses `shift` on the `$@` array on each recognized argument. If all arguments are valid, `-z $@` will evaluate as true because the length of `"$@"` is zero, thus printing the help test incorrectly (resulting in non-interactive mode essentially being non-functional).

> Idea was to show usage on calling the script without parameters or with -h or --help flags.

Ah, that does makes sense. I'll amend this to also imply `--help` when no parameters are passed to cover that case as well.

Reply:
@shkoliar The loop immediately prior to this condition uses `shift` on the `$@` array on each recognized argument. If all arguments are valid, `-z $@` will evaluate as true because the length of `"$@"` is zero, thus printing the help test incorrectly (resulting in non-interactive mode essentially being non-functional).

> Idea was to show usage on calling the script without parameters or with -h or --help flags.

Ah, that does makes sense. I'll amend this to also imply `--help` when no parameters are passed to cover that case as well.

Reply:
@shkoliar Updated. See commit 1d36347